x-pack/test/observability_ai_assistant_api_integration/tests/knowledge_base/knowledge_base_user_instructions.spec.ts

```diff
@@ -91,59 +92,64 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       });
 
       it('"editor" can retrieve their own private instructions and the public instruction', async () => {
-        const res = await observabilityAIAssistantAPIClient.editorUser({
-          endpoint: 'GET /internal/observability_ai_assistant/kb/user_instructions',
+        await retry.try(async () => {
```

Comment:
Can we remove the `retry` call here and re-run the flaky tests? If we can do without it, that would be my preference, otherwise we'll end up chasing flakiness everywhere (there is already a retry at the test suite level, for instance).

Reply:
Thanks for the review @dgieselaar 

Apologies if this is a dumb question. 
How do I re-run the flaky tests?

Reply:
I'll send you a link!